### DISCUSSION TITLE
Socket.io, real time

### DISCUSSION THREAD

**Discussion by FrEtienne** [directus/directus#13947] [Feature Requests]
Real time update front ui when data change in database with socket.io is possible ?

**Comment by rijkvanzanten** [comment]
You can add a custom endpoint to spin up a websocket connection and then rely on event hooks to submit messages. Not trivial to do, so if you're looking for a native solution, give https://github.com/directus/directus/discussions/3221 an upvote 🙂

**Comment by rijkvanzanten** [comment]
Closing as duplicate of https://github.com/directus/directus/discussions/3221
